test/acceptance/test_cwe476.py

```diff
@@ -8,29 +8,35 @@ def setUp(self):
         self.target = '476'
         self.string = b'NULL Pointer Dereference'
 
-    def test_cwe476_01_arm(self):
+    def test_cwe476_01_arm_gcc(self):
         expect_res = 1
-        res = cwe_checker_testlib.execute_and_check_occurence(self.target, self.target, 'arm', self.string)
+        res = cwe_checker_testlib.execute_and_check_occurence(self.target, self.target, 'arm', 'gcc', self.string)
         self.assertEqual(res, expect_res)
 
-    def test_cwe476_01_x86(self):
+    def test_cwe476_01_x86_gcc(self):
         expect_res = 1
-        res = cwe_checker_testlib.execute_and_check_occurence(self.target, self.target, 'x86', self.string)
+        res = cwe_checker_testlib.execute_and_check_occurence(self.target, self.target, 'x86', 'gcc', self.string)
         self.assertEqual(res, expect_res)
 
-    def test_cwe476_01_x64(self):
+    def test_cwe476_01_x64_gcc(self):
         expect_res = 1
-        res = cwe_checker_testlib.execute_and_check_occurence(self.target, self.target, 'x64', self.string)
+        res = cwe_checker_testlib.execute_and_check_occurence(self.target, self.target, 'x64', 'gcc', self.string)
+        self.assertEqual(res, expect_res)
+
+    @unittest.skip("FIXME")
```

Comment:
I should take a look at this.

Reply:
clang seems to optimze away the relevant calls for the check. Is there a "do not optimize anything"-flag for clang that we could use?

Reply:
we could build the binaries with -o0

Reply:
If it is not enabled by default, we should do that.